lib/repl.js

```diff
@@ -158,6 +160,8 @@ function REPLServer(prompt,
     options = {};
   }
 
+  instances++;
```

Comment:
I think I’d prefer it if we could do this only for the built-in REPL rather than counting instances?

Reply:
Done